src/components/templates/Camp/components/Map.tsx

```diff
@@ -269,8 +269,8 @@ export const Map: React.FC<MapProps> = ({
         gridTemplateRows: `repeat(${templateRows}, 1fr)`,
       }}
     >
-      {showGrid && mapGrid}
-      {showGrid && partygoersOverlay}
+      {mapGrid}
+      {partygoersOverlay}
```

Comment:
This is fine because the hooks are already using showGrid and handling this for us. This was a redundant check.